recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   merge_build_host: true  # [win]
-  number: 1
+  number: 2
```

Comment:
just as a question, do you need to bump? only the ppc should be build if you leave it as 1, left?

Reply:
True. We could not bump. There is not a lot of harm here to bump, but I am happy to remove the bump too. I do it by habit now.

Reply:
all good, was just wondering